sampleChangerApp/src/converter.cpp

```diff
@@ -140,9 +136,8 @@ void converter::loadSlotDetails(const char* fname)
 }
 
 // Extract slot details from the xml
-void converter::loadSlotDetails(TiXmlHandle &hRoot)
+std::map<std::string, slotData> converter::loadSlotDetails(TiXmlHandle &hRoot)
```

Comment:
std::unordered_map is much faster and is more commonly used I believe, if we care about order ignore this comment

Reply:
have committed this, feel free to revert, i dont think it matters either way i just remember at a conference that unordered_map is much quicker but takes up slightly more memory (it's a hash map essentially)